docs/src/main/resources/news/2018-07-26-first-release.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: news
+title: First release
+date: 2018-07-26 08:00:00
+section: news
+author:
+name: Jeroen Laverman
+id: laverman
+---
+{% include base.html %}
+
+# Eclipse hawkBit 0.2.0 - First Release
+
+hawkBit is a domain independent back-end framework for rolling out software updates to constrained edge devices as well 
+as more powerful controllers and gateways connected to IP based networking infrastructure. It is part of the Eclipse IoT 
+since 2015 and with _0.2.0_ a first release is available.
+
+In this article, we want to give an overview over the latest highlights of hawkBit and let you know how you can get 
+started in seconds.
+
+## Finally, it is here! 
+
+After being around in the Eclipse IoT realm for quite some time now, we are more than happy to announce our first release: 
+_Eclipse hawkBit 0.2.0_. The release can be found on [Maven Central](https://mvnrepository.com/artifact/org.eclipse.hawkbit) 
+and [Docker Hub](https://hub.docker.com/r/hawkbit/hawkbit-update-server/). It includes the following core features:
+
+* Device and Software Repository
+* Update Management
+* Artifact Content Delivery
+* Rollout Management
+
+which are accessible via the following interfaces:
+
+* Management UI
+* Management API
+* Direct Device Integration (DDI) API
+* Device Management Federation (DMF) API
+
+![hawkBit Overview](../img/hawkBit_overview.jpeg)
+
+
+## What's new?
+
+Whenever there is a new release, the first question that comes to mind is: What's new? Since this is our first release, 
+one could argue that everything is new. However, most of the features are already well established. This holds true, for 
+example, for our APIs or the Rollout Management. Nevertheless, there have been some recent updates to hawkBit, which we 
+do not want to leave unmentioned: 
+
+### Streamlined UI
+
+The probably most noticeable change has been the removal of the two buttons (`Drop here to delete` and `Actions`) at the 
+bottom of the _Deployment_, _Distributions_, and _Upload_ view. This is a major usability improvement! For example, 
+deleting an item required (1) dragging an item onto the delete button, (2) opening the delete pop-up, and (3) confirming 
+the deletion. Now, an item can be easily removed by clicking on its remove icon and confirming the action. This is not 
+only faster and more intuitive, it also saves a lot of display real estate which can now be used to focus on what is 
+important. We hope you like this change as much as we do! _(Requires: hawkBit > 0.2.2)_
+
+![Screenshot of improved UI](../img/hawkbit_ui.png)
+
+### MS SQL Server
+
+Eclipse hawkBit supports a range of different SQL databases. Up to now these were the internal H2 database (which can be 
```

Comment:
they are still supported... therefore I would use "have been" instead of "were"...